### ISSUE TITLE
Can you explain the meaning of your datasets?

### ISSUE THREAD

**Issue by chenchongthu** [zoulixin93/NICF#3]
E.g., which dataset it is (Movielens, EachMovie, or Netflix?),  and how to split it into training set and test set as in your sigir paper?
